x-pack/test/functional/apps/machine_learning/anomaly_detection/categorization_job.ts

```diff
@@ -0,0 +1,392 @@
+      await ml.jobWizardCommon.assertEventRateChartExists();
+      await ml.jobWizardCommon.assertEventRateChartHasData();
+    });
+
+    it('job creation displays the pick fields step', async () => {
+      await ml.jobWizardCommon.advanceToPickFieldsSection();
+    });
+
+    it(`job creation selects ${detectorTypeIdentifier} detector type`, async () => {
+      await ml.jobWizardCommon.assertCategorizationDetectorTypeSelectionExists();
+      await ml.jobWizardCommon.selectCategorizationDetectorType(detectorTypeIdentifier);
+      await ml.jobWizardCommon.assertCategorizationDetectorTypeSelected(detectorTypeIdentifier);
+    });
+
+    it(`job creation selects the categorization field`, async () => {
+      await ml.jobWizardCommon.assertCategorizationFieldInputExists();
+      await ml.jobWizardCommon.selectCategorizationField(categorizationFieldIdentifier, true);
+      // await ml.jobWizardCommon.assertCategorizationFieldSelection([categorizationFieldIdentifier]);
+      await ml.jobWizardCommon.assertCategorizationExamplesCallout(
+        CATEGORY_EXAMPLES_VALIDATION_STATUS.VALID
+      );
+      await ml.jobWizardCommon.assertCategorizationExamplesTable(categorizationExampleCount);
+    });
+
+    it('job creation inputs the bucket span', async () => {
+      await ml.jobWizardCommon.assertBucketSpanInputExists();
+      await ml.jobWizardCommon.setBucketSpan(bucketSpan);
+    });
+
+    it('job creation displays the job details step', async () => {
+      await ml.jobWizardCommon.advanceToJobDetailsSection();
+    });
+
+    it('job creation inputs the job id', async () => {
+      await ml.jobWizardCommon.assertJobIdInputExists();
+      await ml.jobWizardCommon.setJobId(jobId);
+    });
+
+    it('job creation inputs the job description', async () => {
+      await ml.jobWizardCommon.assertJobDescriptionInputExists();
+      await ml.jobWizardCommon.setJobDescription(jobDescription);
+    });
+
+    it('job creation inputs job groups', async () => {
+      await ml.jobWizardCommon.assertJobGroupInputExists();
+      for (const jobGroup of jobGroups) {
+        await ml.jobWizardCommon.addJobGroup(jobGroup);
+      }
+      await ml.jobWizardCommon.assertJobGroupSelection(jobGroups);
+    });
+
+    it('job creation opens the additional settings section', async () => {
+      await ml.jobWizardCommon.ensureAdditionalSettingsSectionOpen();
+    });
+
+    it('job creation adds a new custom url', async () => {
+      await ml.jobWizardCommon.addCustomUrl({ label: 'check-kibana-dashboard' });
+    });
+
+    it('job creation assigns calendars', async () => {
+      await ml.jobWizardCommon.addCalendar('wizard-test-calendar');
+    });
+
+    it('job creation opens the advanced section', async () => {
+      await ml.jobWizardCommon.ensureAdvancedSectionOpen();
+    });
+
+    it('job creation displays the model plot switch', async () => {
+      await ml.jobWizardCommon.assertModelPlotSwitchExists();
```

Comment:
We should also assert the default setting of the model plot switch:
```
      await ml.jobWizardCommon.assertModelPlotSwitchCheckedState(false);
```
In addition, it would be good to assert that in case of the `rare` detector, the model plot switch is disabled.